_infra/helm/locust/values.yaml

```diff
@@ -22,6 +22,7 @@ locust:
       GOOGLE_CLOUD_PROJECT: "ras-rm-performance-20220908"
       GCS_BUCKET_NAME: "ras-rm-performance-20220908-locust"
       CSRF_ENABLED: true
+      json_requests_filepath: _infra/helm/locust/locustfiles/requests.json
```

Comment:
I've not had chance to test this in a Helm deployment in GCP (so this may work!) but is this file path relative to the Locust Python script's current location? i.e. will it try any load
```
_infra/helm/locust/locustfiles/_infra/helm/locust/locustfiles/requests.json
```

Reply:
I've updated how the file is retrieved so that it uses the absolute path of the file rather than the relative one.

Reply:
Were you able to successfully deploy this helm chart to the performance environment? I see the following using the default values.yaml
```
textPayload: "Traceback (most recent call last):
  File "/opt/venv/bin/locust", line 8, in <module>
    sys.exit(main())
  File "/opt/venv/lib/python3.10/site-packages/locust/main.py", line 75, in main
    docstring, _user_classes, shape_class = load_locustfile(_locustfile)
  File "/opt/venv/lib/python3.10/site-packages/locust/util/load_locustfile.py", line 58, in load_locustfile
    imported = source.load_module()
  File "<frozen importlib._bootstrap_external>", line 548, in _check_name_wrapper
  File "<frozen importlib._bootstrap_external>", line 1063, in load_module
  File "<frozen importlib._bootstrap_external>", line 888, in load_module
  File "<frozen importlib._bootstrap>", line 290, in _load_module_shim
  File "<frozen importlib._bootstrap>", line 719, in _load
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/mnt/locust/locustfile.py", line 33, in <module>
    with open(os.path.basename(requests_filepath), encoding='utf-8') as requests_file:
FileNotFoundError: [Errno 2] No such file or directory: 'requests.json'"
timestamp: "2024-03-06T08:53:13.898561775Z"
}
```

Reply:
I've fixed the filepath so that now, helm deployments of this branch are possible.

Reply:
Resolving this as its superseded by the comment below about the /mnt/locust in the configMap